cf-deployment.yml

```diff
@@ -1396,11 +1396,11 @@ instance_groups:
         tls_pem:
         - cert_chain: "((router_ssl.certificate))"
           private_key: "((router_ssl.private_key))"
-        ca_certs: |
-          ((diego_instance_identity_ca.ca))
-          ((cc_tls.ca))
-          ((uaa_ssl.ca))
-          ((network_policy_server_external.ca))
+        ca_certs:
+          - ((diego_instance_identity_ca.ca))
```

Comment:
Can you please fix the alignment of the array such that the hyphen is flush with the key above.

Pretty semantic, but we need to stick with our standards 😁